ouroboros-consensus/bench/mempool-bench/Main.hs

```diff
@@ -129,18 +132,14 @@ main = withStdTerminalHandles $ do
   Adding TestBlock transactions to a mempool
 -------------------------------------------------------------------------------}
 
-openMempoolWithCapacityFor :: [MempoolCmd TestBlock] ->  IO (MockedMempool IO TestBlock)
-openMempoolWithCapacityFor cmds =
-    Mocked.openMockedMempool capacityRequiredByCmds
+openMempoolWithCapacity :: ByteSize ->  IO (MockedMempool IO TestBlock)
+openMempoolWithCapacity capacity =
+    Mocked.openMockedMempool (Mempool.mkCapacityBytesOverride  capacity)
                              Tracer.nullTracer
-                             TestBlock.txSize
                              Mocked.MempoolAndModelParams {
                                  Mocked.immpInitialState = TestBlock.initialLedgerState
                                , Mocked.immpLedgerConfig = TestBlock.sampleLedgerConfig
                              }
-  where
```

Comment:
I moved this whole size calculation (which caused the regression failure in this PR's CI) into the `withResource` stage --- ie "out" of "the benchmark".

What do you think @dnadales ? Was it intentionally being measured, or no?

Reply:
Superseded by #1227